Jenkinsfile

```diff
@@ -297,9 +297,9 @@ pipeline {
               agent {
                 docker {
                   label "linux-cpu-node"
-                  image "nvcr.io/nvidia/pytorch:22.04-py3"
-                  args "-u root"
-                  alwaysPull false
+                  image "rapidsai/cugraph_nightly_torch-cuda:11.5-base-ubuntu18.04-py3.9-pytorch1.12.0-rapids22.10"
+                  args "-u root --runtime nvidia"
+                  alwaysPull true
```

Comment:
What does `label "linux-cpu-node` provision ? Is it a CPU node or a GPU node ? 

If it is a CPU node then we dont need to add `--runtime nvidia`  ? 

Can we try with label `linux-gpu-node` ?

Reply:
My current guess based on looking at the logs is that we fail in the docker run  command  (because we provide `--runtime nvidia` on a CPU node). 

```
docker run -t -d -u 0:0 -u root --runtime nvidia -w /root/jenkins/workspace/dgl_PR-4573@3 -v /root/jenkins/workspace/dgl_PR-4573@3:/root/jenkins/workspace/dgl_PR-4573@3:rw,z -v /root/jenkins/workspace/dgl_PR-4573@3@tmp:/root/jenkins/workspace/dgl_PR-4573@3@tmp:rw,z -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** rapidsai/cugraph_nightly_torch-cuda:11.5-base-ubuntu18.04-py3.9-pytorch1.11.0-rapids22.10 cat
[Pipeline] // withDockerContainer
[Pipeline] }
[Pipeline] // withEnv
[Pipeline] }
[Pipeline] // node
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
Failed in branch PyTorch Cugraph GPU Build
```

If we look at other successful runs  we always see a docker top command follow a docker run command which tells me that it failed at docker run which just does a `cat` which i dont think should fail.   

```
docker run 
 docker top 
```

Reply:
Ah, you're right I think. thanks